client/my-sites/sharing/connections/services/google-my-business.js

```diff
@@ -25,40 +29,77 @@ export class GoogleMyBusiness extends SharingService {
 	};
 
 	// override `createOrUpdateConnection` to ignore connection update, this is only useful for publicize services
-	createOrUpdateConnection = () => {};
+	createOrUpdateConnection = ( keyringConnectionId, externalUserId = 0 ) => {
+		this.props
+			.saveSiteSettings( this.props.siteId, {
+				google_my_business_keyring_id: keyringConnectionId,
+				google_my_business_location_id: externalUserId,
+			} )
+			.then( ( { updated } ) => {
+				if (
+					! updated.hasOwnProperty( 'google_my_business_keyring_id' ) &&
+					! updated.hasOwnProperty( 'google_my_business_location_id' )
+				) {
+					this.props.failCreateConnection( {
+						message: this.props.translate( 'Error while linking your site to %(service)s.', {
```

Comment:
:information_source: This string is probably long enough that it doesn't need a **context**. Are you sure you don't want to use a translator comment instead?